pkg/orm/mock.go

```diff
@@ -2,146 +2,37 @@ package orm
 
 import (
 	"context"
-	"errors"
-	"fmt"
-	"strings"
+	"time"
 
-	sqle "github.com/dolthub/go-mysql-server"
-	"github.com/dolthub/go-mysql-server/memory"
-	gsvr "github.com/dolthub/go-mysql-server/server"
-	gsql "github.com/dolthub/go-mysql-server/sql"
-	"github.com/dolthub/go-mysql-server/sql/information_schema"
-	"github.com/hanfei1991/microcosm/pkg/meta/metaclient"
-	"github.com/hanfei1991/microcosm/pkg/tenant"
-	"github.com/phayes/freeport"
-	"github.com/pingcap/failpoint"
+	cerrors "github.com/hanfei1991/microcosm/pkg/errors"
 	"github.com/pingcap/log"
-	"github.com/pingcap/tiflow/pkg/retry"
+	"go.uber.org/zap"
+	"gorm.io/driver/sqlite" // Sqlite driver based on GGO
```

Comment:
What is `GGO` short for, or is it typo of `CGO`?

Reply:
Already remove the comment.